authors/apps/articles/views.py

```diff
@@ -1,21 +1,4 @@
-from rest_framework import generics
-from rest_framework.permissions import IsAuthenticated
-from .models import Article, ArticleLikes, Bookmark, ArticleRating
-from django.shortcuts import get_object_or_404
-from .serializers import (
-    ArticleSerializer, ArticleUpdateSerializer, BookmarksSerializer,
-    ArticleRatingSerializer)
-from .renderers import ArticleJSONRenderer, BookmarkJSONRenderer
-from rest_framework import status, serializers
-from rest_framework.response import Response
-from authors.apps.utils.messages import error_messages
-from authors.apps.profiles.models import Profile
-from authors.apps.utils.custom_permissions.permissions import (
-    check_if_is_author
-)
-from .paginators import ArticleLimitOffSetPagination
-from .utils import get_like_status, get_usernames
-from rest_framework.exceptions import NotFound
+from .import_statements import *
```

Comment:
@armstrongsouljah undo the changes

Reply:
Let me do that. However, we had opted to do this to avoid the 255 max limit. But let me put them back to how they were. Thank you.